lightning_pose_app/ui/extract_frames.py

```diff
@@ -28,10 +32,12 @@
     export_frames,
     select_frame_idxs_kmeans,
     select_frame_idxs_model,
+    find_contextual_frames,
```

Comment:
for some reason this import dosent work which is why I have the function here in the flow as well